<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="the_image" name="Image_name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>The_Image</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>801</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QLineEdit" name="Iterations">
       <property name="text">
        <string>300</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Resolution:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="CorX">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QLineEdit" name="RWidth">
       <property name="text">
        <string>500</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="Scale_multiplier">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QLineEdit" name="RHeight">
       <property name="text">
        <string>400</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Scale_multiplier    10^</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Scale    + - to sides</string>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Zoom_speed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="11">
      <widget class="QLineEdit" name="Zoom_speed">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="12" rowspan="2">
      <widget class="QLabel" name="Coords">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="Scale">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="CorY">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="ToJulia">
       <property name="text">
        <string>J</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="ToSave">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="JulC_x">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="JulC_y">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>the_image</class>
   <extends>QLabel</extends>
   <header location="global">the_image.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
